test/unit/svn_commits_test.rb

```diff
@@ -110,7 +110,7 @@ def test_remove_dupes_with_sql_acii_encoding
            #Note: If there are two files that are identical except for the encoding, ohloh_scm should only keep one.
            svn = SvnAdapter.new
            diff_1 = OhlohScm::Diff.new(:action => "M", :path => "/FritzBoxDial/FritzBoxDial/formWählbox.vb")
-           diff_2 = OhlohScm::Diff.new(:action => "A", :path => "/FritzBoxDial/FritzBoxDial/formW\xC3\xA4hlbox.vb".force_encoding('ASCII'))
+           diff_2 = OhlohScm::Diff.new(:action => "A", :path => "/FritzBoxDial/FritzBoxDial/formWählbox.vb")
```

Comment:
This doesn't verify the scenario, does it?  There are two different actions, "Modify" and "Add", but the same file, "/FritzBoxDial/FritzBoxDial/formWählbox.vb" without any difference in representation -- that is, both are UTF8, whereas the removed line used Unicode encoding, which is what we saw in the code.